cmd/guacone/cmd/patch.go

```diff
@@ -0,0 +1,397 @@
+					Digest:    "1testArtifactDigest",
+				}},
+				HasSlsa: &model.SLSAInputSpec{
+					BuildType:   "testBuildType",
+					SlsaVersion: "testSlsaVersion",
+					SlsaPredicate: []model.SLSAPredicateInputSpec{
+						{Key: "slsa.testKey", Value: "testValue"},
+					},
+				},
+			},
+		},
+		PointOfContact: []assembler.PointOfContactIngest{
+			{
+				Pkg: &model.PkgInputSpec{
+					Type:      "1pkgType",
+					Namespace: ptrfrom.String("1pkgNamespace"),
+					Name:      "1pkgName",
+					Version:   ptrfrom.String("1.19.0"),
+				},
+				PkgMatchFlag: model.MatchFlags{
+					Pkg: model.PkgMatchTypeSpecificVersion,
+				},
+				PointOfContact: &model.PointOfContactInputSpec{
+					Email:         "testEmail1",
+					Info:          "testInfo",
+					Since:         tm,
+					Justification: "testJustification",
+					Origin:        "testOrigin",
+					Collector:     "testCollector",
+				},
+			},
+			{
+				Pkg: &model.PkgInputSpec{
+					Type:      "2pkgType",
+					Namespace: ptrfrom.String("2pkgNamespace"),
+					Name:      "2pkgName",
+					Version:   ptrfrom.String("1.19.0"),
+				},
+				PkgMatchFlag: model.MatchFlags{
+					Pkg: model.PkgMatchTypeSpecificVersion,
+				},
+				PointOfContact: &model.PointOfContactInputSpec{
+					Email:         "testEmail2",
+					Info:          "testInfo",
+					Since:         tm,
+					Justification: "testJustification",
+					Origin:        "testOrigin",
+					Collector:     "testCollector",
+				},
+			},
+		},
+	}
+)
+
+var queryPatchCmd = &cobra.Command{
+	Use:   "patch plan [flags] purl",
+	Short: "query which packages are affected by the vulnerability associated with specified packageName or packageVersion",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := logging.WithLogger(context.Background())
+		logger := logging.FromContext(ctx)
+
+		opts, err := validateQueryPatchFlags(
+			viper.GetString("gql-addr"),
+			viper.GetString("start-purl"),
+			viper.GetString("stop-purl"),
+			viper.GetInt("search-depth"),
+			viper.GetBool("sample-data"),
+			args,
+		)
+
+		if err != nil {
+			fmt.Printf("unable to validate flags: %s\n", err)
+			_ = cmd.Help()
+			os.Exit(1)
+		}
+
+		httpClient := http.Client{}
+		gqlClient := graphql.NewClient(opts.graphqlEndpoint, &httpClient)
+
+		var startID string
+		var stopID *string
+		stopID = nil
+		if opts.sampleData {
+			err = analysis.IngestTestData(ctx, gqlClient, sampleGraph)
+
+			if err != nil {
+				fmt.Printf("error ingesting test data: %s\n", err)
+			}
+			getPackageIDsValues, err := analysis.GetPackageIDs(ctx, gqlClient, ptrfrom.String("3pkgType"), "3pkgNamespace", "3pkgName", ptrfrom.String("1.19.0"), true, false)
+
+			startID = *getPackageIDsValues[0]
+			if err != nil {
+				logger.Errorf("error get start pkg for simple data: %s", err)
+			}
+
+		} else {
+			startID, err = getPkgID(ctx, gqlClient, opts.startPurl)
+
+			if err != nil {
+				logger.Errorf("error getting start pkg from purl inputted %s \n", err)
```

Comment:
you can just use `logger.Fatalf` so exist right away without needing `os.Exit` after